### PR TITLE
feat: update orioledb base image and s3 config script

### PR DESCRIPTION
**Pull request by sweatybridge**
s3 config is not fully working https://supabase.slack.com/archives/C0625QCGRGF/p1701701032468969?thread_ts=1699952553.029659&cid=C0625QCGRGF

But enabling orioledb locally for docker volume is working fine so we can go ahead with a new release.

### CODE REVIEW

**Review by pcnc — APPROVED** [review]

File: .github/workflows/dockerhub-release-orioledb.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - develop
+      - s3-config
```
Comment (darora) [inline .github/workflows/dockerhub-release-orioledb.yml]: remove if ready for merge

File: common.vars.pkr.hcl [hunk 2]
```diff
@@ -1 +1 @@
-postgres-version = "15.1.0.136"
+postgres-version = "15.1.0.137-rc2"
```
Comment (darora) [inline common.vars.pkr.hcl]: We're no longer merging suffixes to develop (check will be fixed today)

**Review by darora — CHANGES_REQUESTED** [review]

File: .github/workflows/dockerhub-release-orioledb.yml (hunk 1 above)
Comment (sweatybridge) [reply .github/workflows/dockerhub-release-orioledb.yml]: Please don't merge. It's blocked on orioledb testing as mentioned in description.

File: common.vars.pkr.hcl [hunk 3]
```diff
@@ -1 +1 @@
-postgres-version = "15.1.0.141"
+postgres-version = "15.1.0.142"
```
Comment (darora) [inline common.vars.pkr.hcl]: This has already been published

File: ansible/files/postgresql_config/supautils.conf.j2 [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
+supautils.privileged_extensions = 'address_standardizer, address_standardizer_data_us, autoinc, bloom, btree_gin, btree_gist, citext, cube, dblink, dict_int, dict_xsyn, earthdistance, fuzzystrmatch, hstore, http, hypopg, insert_username, intarray, isn, ltree, moddatetime, pg_cron, pg_graphql, pg_hashids, pg_jsonschema, pg_net, pg_stat_monitor, pg_stat_statements, pg_tle, pg_trgm, pg_walinspect, pgaudit, pgcrypto, pgjwt, pgroonga, pgroonga_database, pgrouting, pgrowlocks, pgsodium, pgtap, plcoffee, pljava, plls, plpgsql, plpgsql_check, plv8, postgis, postgis_raster, postgis_sfcgal, postgis_tiger_geocoder, postgis_topology, postgres_fdw, refint, rum, seg, sslinfo, supabase_vault, supautils, tablefunc, tcn, timescaledb, tsm_system_rows, tsm_system_time, unaccent, uuid-ossp, vector, wrappers, orioledb'
```
Comment (darora) [inline ansible/files/postgresql_config/supautils.conf.j2:4]: I'm assuming this will not cause issues if the extension does not exist?

**Review by darora — APPROVED** [review]

File: ansible/files/postgresql_config/supautils.conf.j2 (hunk 4 above)
Comment (sweatybridge) [reply ansible/files/postgresql_config/supautils.conf.j2:4]: Yup we have [test coverage](https://github.com/supabase/postgres/tree/develop/migrations/tests/extensions) for these cases, ie. enable all extensions with supautils installed.

**Review by pashkinelfe — APPROVED** [review]

**Pull request merged**